setup.py

```diff
@@ -23,7 +23,7 @@ def read(*parts):
     url='http://github.com/praekelt/django-google-analytics',
     packages=find_packages(),
     install_requires=[
-        'Django>=1.11,<2.2',
+        'Django<=2.2.5,<2.3',
```

Comment:
are these arrows correct? this is saying less than or equal to 2.2.5 but also less than 2.3?

Reply:
rookie mistake 😄  fixing